portal-impl/src/com/liferay/portal/service/impl/ResourcePermissionLocalServiceImpl.java

```diff
@@ -1201,6 +1210,98 @@ public void setResourcePermissions(
 			companyId, name, scope, primKey, 0, roleIdsToActionIds);
 	}
 
+	@Override
+	public void updateAction(
+			Role role, String selResource, String actionId, int scope,
+			String[] groupIds)
```

Comment:
This set of parameters is not consistent with other methods in this service. Instead of selResource it should be name. Instead of a Role it should have a roleId. Also the order should be consistent with other methods.